source/_integrations/zwave_js.markdown

```diff
@@ -705,10 +705,30 @@ You can also keep track of the roadmap for the Z-Wave integration [here](https:/
 
 #### Which Z-Wave controller should I buy?
 
-Z-Wave supports all known 500 and 700 series Z-Wave controllers. If you are just starting out, we recommend that you purchase a 700 series controller (with firmware updated to >=7.17.2).
+Z-Wave supports all known 500 and 700 series Z-Wave controllers. If you are just starting out, we recommend that you purchase a 700 series controller (with firmware updated to >=7.17.2). When buying a USB controller, verify that its firmware supports backup and restore of its "Non Volatile Memory" (NVM). This allows to migrate a running Z-Wave installation to another controller also supporting backup/restore of NVM. Z-Wave controller firmware may be updateable by Silicon Labs Developer software.
 
 For more information, see [Supported Z-Wave dongles](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules)
 
+#### Z-Wave controller NVM
+
+Z-Wave supports backup and restore of Z-Wave controller "Non Volatile Memory" (NVM) via "Control Panel" -> "Advanced Actions". Backup will save an NVM file into "Store" -> "backups" -> "nvm", where it will automatically be saved into system backups and can also be downloaded. Restore will ask the user to restore a file from the local machine.
```

Comment:
Looks like you're describing functionality of Z-Wave JS UI. The integration doesn't yet support any NVM features, so we shouldn't document that here yet.